Another docker error

Last lines after `docker build -t "scoreboard" .`:

```
...
Step 4/12 : RUN pip install -r requirements.txt                                                                                                                                                 
 ---> Running in de5bb2ce8416
...
copying src/pylibmc/__main__.py -> build/lib.linux-x86_64-2.7/pylibmc

running build_ext

building '_pylibmc' extension

creating build/temp.linux-x86_64-2.7

creating build/temp.linux-x86_64-2.7/src

x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -fno-strict-aliasing -D_FORTIFY_SOURCE=2 -g -fstack-protector-strong -Wformat -Werror=format-security -fPIC -DUSE_ZLIB -I/usr/include/python2.7 -c src/_pylibmcmodule.c -o build/temp.linux-x86_64-2.7/src/_pylibmcmodule.o -fno-strict-aliasing -std=c99

In file included from src/_pylibmcmodule.c:34:0:

src/_pylibmcmodule.h:41:20: fatal error: Python.h: No such file or directory

 #include <Python.h>

                    ^

compilation terminated.

error: command 'x86_64-linux-gnu-gcc' failed with exit status 1

----------------------------------------
Cleaning up...
Command /usr/bin/python -c "import setuptools, tokenize;__file__='/tmp/pip-build-j2iB2V/pylibmc/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-4ZT9_H-record/install-record.txt --single-version-externally-managed --compile failed with error code 1 in /tmp/pip-build-j2iB2V/pylibmc
Storing debug log for failure in /root/.pip/pip.log
The command '/bin/sh -c pip install -r requirements.txt' returned a non-zero code: 1
```

Related error: #148, dockerfile need refactoring

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,8 +1,9 @@
-FROM debian:jessie
+FROM debian:buster
 
 RUN apt-get update && apt-get install -y \
     nginx \
     python \
+    python-dev \
     python-pip \
     supervisor \
     uwsgi \
@@ -17,6 +18,7 @@ COPY doc/nginx.conf /etc/nginx/sites-enabled/default
 COPY doc/docker/supervisord.conf /etc/supervisor/conf.d/
 
 COPY . /opt/scoreboard
+COPY config.example.py /opt/scoreboard/config.py
 WORKDIR /opt/scoreboard
 
 RUN python main.py createdb

